tests/ClientTest.php

```diff
@@ -2528,9 +2516,7 @@ public function testOurSerializer($after_creation)
                     ],
                 ]
             );
-            $events = $client->getSentEvents();
-            $event = array_pop($events);
-            $this->assertEquals(Dummy_Serializer::$default_value, $event['extra']['foo']);
+            $serializer->__phpunit_verify();
```

Comment:
As we're not testing anymore the return value of the serializer, the whole `foreach` block isn't needed. Just capture a single message and check that the `serialize` method of the mocked object is called